doc/nrf/gs_modifying.rst

```diff
@@ -150,9 +220,29 @@ See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how
 To locate a specific configuration option, use the filter (:guilabel:`Jump to` in menuconfig and guiconfig).
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
-.. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+Changing the configuration permanently
+======================================
+
+To configure your application and maintain the configuration when you clean the build directory, you need to specify the configuration in one of the permanent configuration files.
+In most cases, this will be either the default :file:`prj.conf` file of the application or an extra Kconfig fragment.
+In these files, you can specify different values for configuration options that are defined by a library or board, and you can add configuration options that are specific to your application.
+
+See :ref:`zephyr:setting_configuration_values` in the Zephyr documentation for information on how to change the configuration permanently.
+
+.. tip::
+   Modifying the temporary configuration can result in an invalid or bloated configuration, because the default values have already been calculated based on the previous configuration.
+   It is a good practice to modify :file:`prj.conf` instead of modifying the configuration in menuconfig.
+
+If you work with SES, the :file:`prj.conf` file is read when you open a project.
+This means that after you edit this file, replace it with the newer version by using the :guilabel:`Project` > :guilabel:`Run CMake...` option.
+
+.. _configuring_vsc:
+
+Configuring in the VS Code extension
+====================================
```

Comment:
should we really have `VS Code extension` as a dedicated section ?

Wouldn't it be better to have a common section on how to adjust CMake build arguments, boards, etc. with different tabs, with a tab for:
- Command line
- SES
- VS Code extension

For example adjust this section:
http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-6121/nrf/gs_modifying.html#providing-cmake-options

Reply:
I created a task for this some days ago: https://projecttools.nordicsemi.no/jira/browse/NCSDK-12765